### PR TITLE
Wait for articles

### PR DESCRIPTION
**Pull request by loui981c**
## Description

Adding a bouncing logo when waiting for the fetched articles from the database
  
## Changes

- Adding bounce animation

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.  (on test fails because the test is broken, the function works as intended. The test have been ignored) 
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)
IMG_1130

## Backend branch
-preds

## Notes for Reviewers

Wait with merge until preds are merged

<3

### CODE REVIEW

File: screens/NewsFeed.jsx [hunk 1]
```diff
@@ -1,50 +1,69 @@
...
+      setWaiting(false);
       
       // Set loading to false after fetching articles, but wait at least 2000ms
-      setTimeout(() => {
-        setLoading(false);
-      }, 2000);
+      if (loading) {
+        setTimeout(() => {
+          setLoading(false);
+        }, 2000);
```
Comment (benceisonline) [inline screens/NewsFeed.jsx:60]: Can we define a loading time constant at the top?

File: screens/NewsFeed.jsx [hunk 2]
```diff
@@ -56,7 +75,8 @@ export default function NewsFeedScreen() {
 
 	const handleLoadMore = async (event) => {
 		const { layoutMeasurement, contentSize, contentOffset } = event.nativeEvent;
-		const isAtBottom = layoutMeasurement.height + contentOffset.y >= contentSize.height;
+    // Fetch more articles when user has scrolled almost to the bottom of the screen
+		const isAtBottom = layoutMeasurement.height + contentOffset.y >= contentSize.height - 500;
```
Comment (benceisonline) [inline screens/NewsFeed.jsx:79]: Commented at other PR - constant for 500 instead

File: utils/AxiosRequest.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
 const axios = require('axios').default;
 
-const localUrl = '192.168.68.102';
+const localUrl = '172.20.10.4';
 const apiUrl = `http://${localUrl}:8000`;
 
-export const fetchPredictions = async (userId, noOfRecommendations) => {
+export const fetchPredictions = async (userId, startIndex, noOfRecommendations) => {
```
Comment (benceisonline) [inline utils/AxiosRequest.js:6]: Shouldn't there also be a backend PR too with updated request model?

Comment (loui981c) [reply utils/AxiosRequest.js:6]: jeps, den ligger i preds branchen. my bad haha

**Pull request merged**